File: app/src/main/java/pers/victor/androiddemos/activity/PTIRecyclerViewActivity.java
package pers.victor.androiddemos.activity;

import java.util.ArrayList;
import java.util.List;

import pers.victor.androiddemos.R;
import pers.victor.androiddemos.view.PTIRecyclerView;

/**
 * Created by Victor on 2017/4/1. (ง •̀_•́)ง
 */

public class PTIRecyclerViewActivity extends ToolbarActivity {
    @Override
    public int bindLayout() {
        return R.layout.activity_pti_recycler;
    }

    @Override
    public void initView() {
        List<String> list = new ArrayList<>();
        list.add("");
        list.add("");
        PTIRecyclerView rv = $(R.id.rv_group);
        rv.setUp(new PTIRecyclerView.Adapter<String>(this, R.layout.rv_pti_item, list) {
            @Override
            public void onBindViewHolder(Holder holder, int position) {
            }
        });
    }
}

File: app/src/main/kotlin/pers/victor/androiddemos/activity/PTIRecyclerViewActivity.kt
package pers.victor.androiddemos.activity

import pers.victor.androiddemos.R
import pers.victor.androiddemos.view.PTIRecyclerView
import kotlinx.android.synthetic.main.activity_pti_recycler.*


/**
 * Created by victor on 16-8-1.
 */
class PTIRecyclerViewActivity : ToolbarActivity() {

    override fun bindLayout() = R.layout.activity_pti_recycler

    override fun initView() {
        val list = arrayListOf<String>()
        for (i in 0..2) {
            list.add("")
        }
        rv_group.setUp(object : PTIRecyclerView.Adapter<String>(this, R.layout.rv_pti_item, list) {
            override fun onBindViewHolder(holder: Holder, position: Int) {
            }
        })
    }
}